src/pages/LoginPage.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import Header from '../components/Header_type_1';
+import Footer from '../components/Footer';
+import Login2FA from '../components/Login2FA';
+import LoginWeb3 from '../components/LoginWeb3';
+import LoginNeuKey from '../components/LoginNeuKey';
+
+import styles from './LoginPage.scss';
+import common from '../styles/common.scss';
+
+export default () =>
+  (<div className={common.appContainer}>
+    <Header />
+    <div className={common.contentArea}>
+      <div className={common.widthLimiter}>
+        <Grid fluid>
+          <Row>
+            <Col lgOffset={1} lg={10}>
+              <Row>
```

Comment:
There should not be a `Row` inside a `Col`. Instead, following Ola's layout, there should by three `Col` instances, all with `lg=3`, the first one will have an offset.